docs/openshift.asciidoc

```diff
@@ -19,30 +19,36 @@ It starts an OpenShift cluster just for you, allowing you to take your first ste
 For setting up Kafka and Kafka Connect on OpenShift, a set of images provided by the http://strimzi.io/[Strimzi] project can be used, which offers "Kafka as a Service".
 It consists of enterprise grade configuration files and images that bring Kafka to OpenShift.
 
-First we install the Kafka broker and Kafka Connect templates into our OpenShift project:
+First we install the controlloers and templates for the Kafka broker and Kafka Connect into our OpenShift project:
 
-[listing,options="nowrap"]
+[listing,subs="attributes",options="nowrap"]
 ----
-oc create -f https://raw.githubusercontent.com/strimzi/strimzi/0.1.0/kafka-statefulsets/resources/openshift-template.yaml
-oc create -f https://raw.githubusercontent.com/strimzi/strimzi/0.1.0/kafka-connect/s2i/resources/openshift-template.yaml
+export STRIMZI_VERSION={strimzi-version}
+git clone -b $STRIMZI_VERSION https://github.com/strimzi/strimzi
```

Comment:
I actually liked the fact that it pulled the files via HTTP instead of cloning the repo. Is that not possible any more? Or what did make you change it?

Reply:
Now it means installing 8 files, check
https://github.com/strimzi/strimzi/tree/0.2.0/examples/templates/cluster-controller
https://github.com/strimzi/strimzi/tree/0.2.0/examples/install/cluster-controller

It seems to me more manageble this way

Reply:
Ok, I see.

Reply:
What not downloading the release from the "releases" and unpacking the tar.gz and then using it for Strimzi ?

Reply:
No strong preference either. Usually I'd prefer getting the distro, but here it doesn't really make a difference as we "just" get some text files, no binaries.